packages/base/src/commands.ts

```diff
@@ -257,11 +257,14 @@ const OPERATORS = {
       const selected = model.localState?.selected.value || {};
       const sel0 = getSelectedMeshName(selected, 0);
       const sel1 = getSelectedMeshName(selected, 1);
+      const baseName = sel0 || objects[0].name || '';
+      const baseModel = model.sharedModel.getObjectByName(baseName);
       return {
         Name: newName('Cut', model),
-        Base: sel0 || objects[0].name || '',
+        Base: baseName,
         Tool: sel1 || objects[1].name || '',
         Refine: false,
+        Color: baseModel?.parameters?.Color,
```

Comment:
Maybe provide the default value if it's not defined
```suggestion
        Color: baseModel?.parameters?.Color || DEFAULT_MESH_COLOR,
```

Reply:
Same comment applies to other

Reply:
On it.